### PR TITLE
Fixed Very Simple Typos in Roadmap.md

### PR DESCRIPTION
**Pull request by KaranSinghBisht**
Fixing really small typos

### CODE REVIEW

File: ROADMAP.md [hunk 1]
```diff
@@ -118,9 +118,9 @@ Typically the items under this category fall under 3 to 6 month roadmap. At a hi
 
 ## Future
 
-As the name suggests this bucket contains items that are planned for future. Some times the items are related to adapting to the changes coming in the Kubernetes repo or other related projects. Some of the items currently planned include:
+As the name suggests this bucket contains items that are planned for future. Sometimes the items are related to adapting to the changes coming in the Kubernetes repo or other related projects. Some of the items currently planned include:
 - Support for working with multiple network interfaces
-- Automate the installation of pre-requisities on Kubernetes nodes like setting up iSCSI or huge page
+- Automate the installation of pre-requisites on Kubernetes nodes like setting up iSCSI or huge page
```
Comment (prateekpandey14) [inline ROADMAP.md:123]: nit: better to change as `prerequisites` ?

**Comment by prateekpandey14** [comment]
@KaranSinghBisht , To close on this PR the commit needs to be signed. can you please sign off the commit with your email id as described here:
https://github.com/openebs/openebs/blob/master/CONTRIBUTING.md#sign-your-work

**Comment by KaranSinghBisht** [comment]
@prateekpandey14 did I do it right? Please let me know 
This is something new

**Comment by prateekpandey14** [comment]
Thanks @KaranSinghBisht  for docs fixes. 
You can sign up for Hacktoberfest swag from MayaData by entering details here: https://github.com/mayadata-io/swag/issues/new/choose

**Pull request merged**